mysql-test/columnstore/basic/r/std_aggregate_columnstore.result

```diff
@@ -0,0 +1,217 @@
+DROP DATABASE IF EXISTS std_test_db;
```

Comment:
The test should be removed b/c it is outside the scope of our control thus any related server change will break the test and we can do nothing with this.